src/seedu/addressbook/ui/TextUi.java

```diff
@@ -29,47 +29,20 @@
 
     private static final String DIVIDER = "===================================================";
 
-    /** Format of indexed list item */
-    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
-
-
-    /** Offset required to convert between 1-indexing and 0-indexing.  */
-    public static final int DISPLAYED_INDEX_OFFSET = 1;
-
-    /** Format of a comment input line. Comment lines are silently consumed when reading user input. */
-    private static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
 
     private final Scanner in;
     private final PrintStream out;
 
+    private Formatter formatter;
```

Comment:
Should I use private final Formatter formatter or just private Formatter formatter?

Reply:
Since there is only one instance of Formatter, final would seem more appropriate =) !